src/app/model/Piece.ts

```diff
@@ -26,7 +26,7 @@ export abstract class Piece {
 
   protected static isWithinChessboardBorders(position: Square): boolean {
     const columnNumber = columns.indexOf(position.column);
-    return columnNumber < BOARDSIZE && columnNumber >= 0 && position.row <= BOARDSIZE && position.row > 0;
+    return columnNumber < BOARD_SIZE && columnNumber >= 0 && position.row <= BOARD_SIZE && position.row > 0;
```

Comment:
Fajna zmiana, ale czy ona na pwno tyczy się tego PR? Wywaliłbym ją do innego taska

Reply:
masz rację, następnym razem może będę robił osobne branche...ale jak jestem w trakcie czegoś i taka pierda wchodzi to po prostu nie chce mi się znowu wszystkiego zapisywać, commitować, odpalać osobnego brancha itd. :-)